tap_hubspot/automation_streams.py

```diff
@@ -0,0 +1,66 @@
+"""Stream type classes for tap-hubspot."""
+# from black import Report
+from math import inf
+import requests
+import singer
+import json
+
+from dateutil import parser
+import datetime, pytz
+import time
+from datetime import datetime
+
+from pathlib import Path
+from typing import Any, Dict, Optional, Union, List, Iterable
+
+from singer_sdk import typing as th  # JSON Schema typing helpers
+from pathlib import Path
+from typing import Any, Dict, Optional, Union, List, Iterable
+
+from memoization import cached
+
+from singer_sdk.helpers.jsonpath import extract_jsonpath
+from singer_sdk.streams import RESTStream
+from singer_sdk.authenticators import BearerTokenAuthenticator
+from singer_sdk import typing as th  # JSON schema typing helpers
+from tap_hubspot.client import HubspotStream
+from tap_hubspot.streams import ContactsStream
```

Comment:
Feels like a lot of imports for this tiny file... Can we clean it up?

Adding a linter to this project would be really nice I think. It's starting to grow big enough that it is worth it to keep the code clean

Reply:
Yeah once I finish this marketing rush this would be a good idea. If you have any suggestions on what linter to implement, I would be super grateful for you to put them in this task :D https://potloc.atlassian.net/browse/DE-247?atlOrigin=eyJpIjoiNzIzMDcxMjUzYmIzNDgwYjlhYzIxODE3YjAzYTNkYjIiLCJwIjoiaiJ9